src/command-line/install.js

```diff
@@ -41,18 +41,22 @@ program
 			// Create node_modules folder, otherwise npm will start walking upwards to find one
 			fsextra.ensureDirSync(packagesPath);
 
-			// Create package.json with private set to true to avoid npm warnings
-			fs.writeFileSync(packagesConfig, JSON.stringify({
-				private: true,
-				description: "Packages for The Lounge. All packages in node_modules directory will be automatically loaded.",
-			}, null, "\t"));
+			// Create package.json with private set to true to avoid npm warnings, if
+			// it doesn't exist already
+			if (!fs.existsSync(packagesConfig)) {
+				fs.writeFileSync(packagesConfig, JSON.stringify({
+					private: true,
+					description: "Packages for The Lounge. All packages in node_modules directory will be automatically loaded.",
+				}, null, "\t"));
+			}
```

Comment:
Without this change, file gets emptied every time, which adds to the issue caused by `--no-save`.